### PR TITLE
docs: fix changelog

### PR DESCRIPTION
**Pull request by pmalek-sumo**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,21 +11,36 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
-- chore: upgrade Fluentd to 1.14.4-sumo-1 [#2057][#2057]
 - chore: update the Telegraf image to 1.21.2 [#2036][#2036]
+- chore: upgrade Fluentd to 1.14.4-sumo-1 [#2057][#2057]
```
Comment (sumo-drosiek) [inline CHANGELOG.md:15]: why this change?

Comment (pmalek-sumo) [reply CHANGELOG.md]: Removed this :) Thanks
